sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/student-loan-balances/student-loan-balances-queue.scheduler.ts

```diff
@@ -0,0 +1,63 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { Job, Queue } from "bull";
+import { BaseScheduler } from "../../base-scheduler";
+import { QueueNames } from "@sims/utilities";
+import { QueueService } from "@sims/services/queue";
+import {
+  InjectLogger,
+  LoggerService,
+  ProcessSummary,
+} from "@sims/utilities/logger";
+import { logProcessSummaryToJobLogger } from "../../../../utilities";
+import { StudentLoanBalancesProcessingService } from "@sims/integrations/esdc-integration";
+import { ProcessResponseQueue } from "../models/esdc.models";
+/**
+ * Process Student Loan Balances file from the SFTP location.
+ */
+@Processor(QueueNames.StudentLoanBalances)
+export class StudentLoanBalancesScheduler extends BaseScheduler<void> {
+  constructor(
+    @InjectQueue(QueueNames.StudentLoanBalances)
+    schedulerQueue: Queue<void>,
+    queueService: QueueService,
+    private readonly studentLoanBalancesProcessingService: StudentLoanBalancesProcessingService,
+  ) {
+    super(schedulerQueue, queueService);
+  }
+
+  /**
+   * Process Student Loan Balances files from the SFTP
+   * and update the database with processed records.
+   * @param job Student Loan Balances job.
+   * @returns processing result.
+   */
+  @Process()
+  async processStudentLoanBalancesFiles(
+    job: Job<void>,
+  ): Promise<ProcessResponseQueue[] | string[]> {
```

Comment:
`ProcessResponseQueue` should not be used.
`ProcessSummary` should be used instead as done in the most recent schedulers, for instance, `assessment-workflow-enqueuer.scheduler.ts` and `assessment-workflow-queue-retry.scheduler.ts`.